### DISCUSSION TITLE
Audio on live stream only?

### DISCUSSION THREAD

**Discussion by ragenhe** [blakeblackshear/frigate#5324] [General]
Is it possible to have audio for the live view only and not in the recordings?

**Comment by NickM-27** [comment]
Which version are you referring to? in any case yes, simply adjust the output_args to disable audio

**Reply by ragenhe** [reply]
Thank you! I'm running the most recent beta.

I will look through the presets and see if I can find one that is no audio.

**Reply by NickM-27** [reply]
`preset-record-generic`

**Reply by ragenhe** [reply]
Thank you!!
